src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -825,3 +825,69 @@
   - name: Get the details for a virtual cluster
     text: az sql virtual-cluster show -g mygroup -n mycluster
 """
+
+helps['sql db classification'] = """
+type: group
+short-summary: Manage sensitivity classifications.
+"""
+
+helps['sql db classification update'] = """
+type: command
+short-summary: Update a columns's sensitivity classification.
+examples:
+  - name: Update sensitivity classification for a given column.
+    text: az sql db classification update -g mygroup -s myserver -n mydb --schema dbo --table mytable --column mycolumn --information-type Name --label-name "Confidential - GDPR"
+"""
+
+helps['sql db classification list'] = """
+type: command
+short-summary: Get the sensitivity classifications of a given database.
+examples:
+  - name: List the sensitivity classification of a given database.
+    text: az sql db classification list -g mygroup -s myserver -n mydb
+"""
+
+helps['sql db classification show'] = """
+type: command
+short-summary: Get the sensitivity classification of a given column.
+examples:
+  - name: Get the sensitivity classification of a given column.
+    text: az sql db classification show -g mygroup -s myserver -n mydb --schema dbo --table mytable --column mycolumn --sensitivity-label-source current
```

Comment:
Why is `--sensitivity-label-source current` needed here? Wouldn't recommended be under `recommendation` group?

Reply:
Correct, I removed the sensitivity-label-source for this.
Regarding recommended, getting one classification is not supported yet, so I'm not adding it for now